### DISCUSSION TITLE
There should be a possibility to check data as "valid" if a participant fails a suspicion check or similar

### DISCUSSION THREAD

**Discussion by storyteller-dev** [ctreffe/alfred#154] [Ideas]
At the end of my experiment, the participant has to answer multiple questions about the video shown during the experiment, in order to check if he or she actually paid attention during said video. 
If all questions are answered correctly, the data is valid an will be included in my analysis. But if this attention check fails, the data is invalid and should be stored, but not be included when the `ListRandomizer` checks how many participants I need until my required sample size is achieved. In other words: I still want to be able to compare valid with invalid data in my discussion, but will only work with valid data when analyzing my hypotheses.

By problem with alfred3 is, that it seems that there's currently no possibility to exclude certain data when working with the `ListRandomizer`. 
If I try to mark data as "aborted", an error will be thrown and if I use the `abort()` method, the experiment will conclude. I could of course put the attention check at the very end of the experiment and alter the abort method, but because the participants should have the possibility to participant in a raffle and enter an email after the experiment is finished, I'd rather not use this workaround.

So would it be a good idea to include a feature that allows marking data as "valid" or "invalid" in order to collect valid data as long as the required sample size is not met and not have the experiment closed too soon?

**Comment by jobrachem** [comment]
I am suprised about your report of an error. Would you describe in detail, how you implemented the "abort" workaround, and share your error message?
